### PR TITLE
fix: show error on Editor composer textarea

### PR DESCRIPTION
**Pull request by wa0x6e**
### Summary

Laying the groundworks for #517 

The current proposal creation form does not validate the body, it just set a hard limit on the body textarea. 

In addition, the composer editor textarea is not styled exactly like the rest of the textarea, preventing using the same css to style the errors.

This PR will:

- add error support for the composer textarea, and validating the proposal body and user space statement
- refactor the editor textarea, to make use of the existing textarea styling

### Test 

- Go to the proposal creation page
- Fill the textarea over the 9600 chars limit
- It should show the error 

Screenshot 2024-07-23 at 23 38 02

The PREVIEW markdown tab default height has also been adjusted to match the textarea height by default

Screenshot 2024-07-23 at 23 38 43

### CODE REVIEW

**Review by Sekhmet — COMMENTED** [review]
If I have textarea with error and switch to markdown preview and back error state disappears.

**Comment by wa0x6e** [comment]
> If I have textarea with error and switch to markdown preview and back error state disappears.

Fixed.

Also added validation to the user space statement

File: apps/ui/src/components/Ui/Composer.vue [hunk 1]
```diff
@@ -74,14 +115,34 @@ const editor = useMarkdownEditor(
...
-        maxlength="9600"
-        class="s-input h-[200px] !rounded-t-none !mb-0 !pt-[15px]"
-        @input="event => (model = (event.target as HTMLInputElement).value)"
-      />
-    </div>
+    <textarea
+      ref="editorRef"
+      v-model.trim="model"
+      class="s-input h-[200px]"
+      @input="event => (model = (event.target as HTMLInputElement).value)"
```
Comment (Sekhmet) [inline apps/ui/src/components/Ui/Composer.vue:122]: As it's now `v-model` we don't need `@input` anymore I think.

**Comment by bonustrack** [comment]
Can we show a different limit on space using Turbo? It should be 40k for Turbo, also for normal spaces I believe the limit should be 10k

**Comment by wa0x6e** [comment]
> Can we show a different limit on space using Turbo? It should be 40k for Turbo, also for normal spaces I believe the limit should be 10k

Yes, but this will come from a different PR #524

**Review by bonustrack — APPROVED** [review]
tACK

**Review by Sekhmet — APPROVED** [review]
tACK

**Pull request merged**